core-java-modules/core-java-arrays-operations-advanced/src/test/java/com/baeldung/arraycompare/ArraysCompareUnitTest.java

```diff
@@ -0,0 +1,29 @@
+package com.baeldung.arraycompare;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+
+class ArraysCompareUnitTest {
+
+    @Test
+    void givenSameContents_whenCompare_thenCorrect() {
+        String[] array1 = new String[] { "A", "B", "C" };
+        String[] array2 = new String[] { "A", "B", "C" };
+
+        assertThat(Arrays.compare(array1, array2)).isEqualTo(0);
+    }
+
+    @Test
+    void givenDifferentContents_whenCompare_thenDifferent() {
+        String[] array1 = new String[] { "A", "B", "C" };
+        String[] array2 = new String[] { "A", "C", "B", "D" };
+
+        assertThat(Arrays.compare(array1, array2) < 0).isTrue();
```

Comment:
There's another way to write these assertions in AssertJ without using the less-than and greater-than operators. They're a bit more readable. For example:
`assertThat(Arrays.compare(array1, array2).isLessThan(0);`